draft-ietf-quic-multipath.md

```diff
@@ -1019,7 +1019,7 @@ is an extension of the ACK frame defined by {{QUIC-TRANSPORT}}. It is
 used to acknowledge packets that were sent on different paths using
 multiple packet number spaces. If the frame type is TBD-01, ACK_MP frames
 also contain the sum of QUIC packets with associated ECN marks received
-on the connection up to this point.
+on the acknowledged path up to this point.
```

Comment:
```suggestion
on the acknowledged packet number space up to this point.
```

"acknowledged path" can be confusing, as it might imply all the packet number spaces  (i.e., that of Initial, Handshake and the first application space) that share the original path. It'd be less confusing to repeat the phrasing of QUIC v1, stating that ECN counters are per-packet number space.